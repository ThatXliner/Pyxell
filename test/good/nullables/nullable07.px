
[Int?] a = [null]
print a

[Bool?] b = [null, true, false]
print b

[Char?] c = ['a', null]
print c
