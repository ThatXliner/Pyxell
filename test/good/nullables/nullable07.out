[null]
[null, true, false]
[a, null]
